install/build.gradle

```diff
@@ -124,15 +124,15 @@ task runApp(type: Exec) {
     }
 
     if (appArgs != null) {
-        args += appArgs
+        args += appArgs//.split("\\s+")
```

Comment:
I'm confused by the comment  //.split(...)

Reply:
oh, when I originally made that change I thought I needed to split the provided string into an array, it turns out not. Forgot to delete the code completely.